runners/google-cloud-dataflow-java/src/main/resources/org/apache/beam/runners/dataflow/dataflow.properties

```diff
@@ -16,6 +16,6 @@
 #
 # Dataflow runtime properties
 
-legacy.environment.major.version=${dataflow.legacy_environment_major_version}
-fnapi.environment.major.version=${dataflow.fnapi_environment_major_version}
-container.version=${dataflow.container_version}
+legacy.environment.major.version=@dataflow.legacy_environment_major_version@
```

Comment:
<!--new_thread; commit:55d77e44c84282dae7e9ca1e86ce09d7744d6ee9; resolved:0-->
This change will make it so mvn and gradle cannot coexist?